### ISSUE TITLE
Python 3 issue

### ISSUE THREAD

**Issue by arigit** [jcsaaddupuy/python-xbmc#9]
I installed this module using pip3 in ubuntu 12.04
sudo pip3 install xbmc-json

The installation completed OK.

However when importing the class I am getting this error:

```
Traceback (most recent call last):
  File "/usr/local/bin/pvr", line 97, in <module>
    from xbmcjson import XBMC, PLAYER_VIDEO
ImportError: cannot import name 'XBMC'
```

Is python3.4 fully supported?

**Comment by Lunatixz** [comment]
no python 3 support has been directly added... what build of Kodi uses python 3?

**Comment by arigit** [comment]
I use kodi on my ubuntu 12.04 htpc. Kodi's addons use python 2.x; I use xbmc-json  (which I converted to kodi-json) inside a python PVR script I wrote and use in my setup. My script is using python 3. 

the problem is that for some reason the command 

sudo pip3 install xmbc-json

completed OK but the XBMC class is unusable for some reason. Perhaps pip3 is downloading some older version and not master.

I worked around the problem by downloading  master (zip) from github and importing the class 'manually' from my script.
